templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -472,6 +480,40 @@ <h2>
     </section>
   </div>
 
+  <hr class="p-rule is-fixed-width" />
+
+  <div class="u-fixed-width">
```

Comment:
You may remove this parent div since row already adds the necessary padding that is otherwise achieved by `u-fixed-width`